src/scribe_data/utils.py

```diff
@@ -240,6 +244,66 @@ def get_language_words_to_ignore(language: str) -> list[str]:
     )
 
 
+def get_language_dir_path(language):
```

Comment:
If this is not used anywhere else in the code, could we rename it to be internal as `_get_language_dir_path`?

Reply:
done, changed to `_get_language_dir_path`.